modules/core/src/main/java/org/apache/synapse/aspects/flow/statistics/collectors/RuntimeStatisticCollector.java

```diff
@@ -95,13 +112,69 @@ public static void init() {
                     eventConsumerTime;
             log.info("Statistics Entry Expiration time set to " + eventExpireTime + " milliseconds");
             new MediationFlowController();
+
+            if (isOpenTracingEnabled) {
+                initOpenTracing(isCollectingPayloads, isCollectingProperties);
+            }
         } else {
             if (log.isDebugEnabled()) {
                 log.debug("Statistics is not enabled in \'synapse.properties\' file.");
             }
         }
     }
 
+    private static void initOpenTracing(boolean isCollectingPayloads, boolean isCollectingProperties) {
+        final String DEFAULT_JAEGER_SAMPLER_MANAGER_HOST = "localhost";
+        final String DEFAULT_JAEGER_SAMPLER_MANAGER_PORT = "5778";
+        final String DEFAULT_JAEGER_SENDER_AGENT_HOST = "localhost";
+        final String DEFAULT_JAEGER_SENDER_AGENT_PORT = "6831";
+        final String DEFAULT_JAEGER_REPORTER_MAX_QUEUE_SIZE = "100";
+        final String DEFAULT_JAEGER_REPORTER_FLUSH_INTERVAL = "10000";
+
+        // Jaeger Sampler Configurations
+        String samplerManagerHost =
+                SynapsePropertiesLoader.getPropertyValue(
+                        StatisticsConstants.JAEGER_SAMPLER_MANAGER_HOST, DEFAULT_JAEGER_SAMPLER_MANAGER_HOST);
+        String samplerManagerPort =
+                SynapsePropertiesLoader.getPropertyValue(
+                        StatisticsConstants.JAEGER_SAMPLER_MANAGER_PORT, DEFAULT_JAEGER_SAMPLER_MANAGER_PORT);
+        String samplerManagerHostPort = samplerManagerHost + ":" + samplerManagerPort;
+
+        // Jaeger Sender Configurations
+        String senderAgentHost =
+                SynapsePropertiesLoader.getPropertyValue(
+                        StatisticsConstants.JAEGER_SENDER_AGENT_HOST, DEFAULT_JAEGER_SENDER_AGENT_HOST);
+        String senderAgentPort =
+                SynapsePropertiesLoader.getPropertyValue(
+                        StatisticsConstants.JAEGER_SENDER_AGENT_PORT, DEFAULT_JAEGER_SENDER_AGENT_PORT);
+
+        // Jaeger Reporter Configurations
+        String logSpans =
+                SynapsePropertiesLoader.getPropertyValue(
+                        StatisticsConstants.JAEGER_REPORTER_LOG_SPANS, String.valueOf(false));
```

Comment:
can use org.apache.synapse.config.SynapsePropertiesLoader#getBooleanProperty no need to parse here since we have utility for this

Let's use that for all boolean property reads

Reply:
Done in https://github.com/wso2/wso2-synapse/pull/1466/commits/40acd3cca2e6a427b0c30c9aab9a262df62034fe